assets/js/modules/adsense/util/status.js

```diff
@@ -56,12 +57,13 @@ export const determineAccountStatus = ( {
 	accounts,
 	clients,
 	alerts,
-	error,
+	accountsError,
+	alertsError,
 	previousAccountID,
 	previousClientID,
 } ) => {
 	if ( undefined === accounts || undefined === previousAccountID ) {
-		return accountsErrorToStatus( error );
+		return accountsErrorToStatus( accountsError );
```

Comment:
@felixarntz should this conditional be updated to perhaps only consider if there is an `accountsError` rather than checking `accounts` and `previousAccountID`?

Reply:
I think it's okay as is - if both of the currently checked variables are defined, let's rely on that and ignore any error - although in practice there shouldn't be any `accountsError` in that circumstance anyway.